snakemake.md

```diff
@@ -31,13 +31,29 @@ or, alternatively, if you prefer `pip` over `conda`:
 $ pip3 install snakemake
 ```
 
-This may fail with an error message containing 
+However, snakemake has non-python dependencies, so a pip-based
+installation has limited functionality unless these dependencies
+are manually installed.
```

Comment:
Can we say if pip-based is enough for us, and also what kind of functionality is missing?

In my own test, I used it through pip and it works fine.  I could see it needed for some other batch system integration and the like, which I'd put into the category "if you need it, you'll know"

Reply:
I am now sending a commit addressing this.